### ISSUE TITLE
Keep track of errors/inconsistencies in the textbook and in the code

### ISSUE THREAD

**Issue by smiths** [smiths/vdisp#27]
The examples where the code and the documentation have errors and or inconsistencies are valuable for future papers.  @EmilSoleymani we should keep track of them as you discover them.

Please create a file where you put all of the examples where you find mistakes/inconsistencies in the code and documentation.  In some cases we might later find that the misunderstanding is ours, but it is best to have these in one place to consult.  We can also ask Dr. Stolle to review our examples.  You've already fixed one typo in the code.  In issue #14 you mention potential issues with the textbook.

You can create the file in whatever file type is most convenient for you.  A wiki entry or as a LaTeX file, would be fine.

**Comment by EmilSoleymani** [comment]
Since the README file has a link to an online pdf of the textbook, where certain errors that were present in our local pdf aren't present, should I only document errors existing in both?

**Comment by EmilSoleymani** [comment]
Indeed the online PDF is from 1990 while the scanned version is from 1994. The only time I have seen the 1994 version have a (potential) error the 1990 version does not is in the ph_ax calculations. Recall that in the 1994 version the formula was given as the following:
<img width="419" alt="image" src="https://user-images.githubusercontent.com/97846411/167464316-e8fd1720-a760-4db6-9185-c15a3f4e2aab.png">
Where phi_px is never defined of mentioned. In both the code and the 1990 version, there is a phi_ps in the place of phi_px:
<img width="401" alt="image" src="https://user-images.githubusercontent.com/97846411/167464472-bd322f8f-ce75-4882-8780-f3cb157012dd.png">
This could be indicating that there is a new way of calculating the phi_ax variable that has not been reflected in the code as opposed to it being a typo? Was our FORTRAN source code copied from the last pages of the textbook or it was given as a file.

**Comment by EmilSoleymani** [comment]
Added inconsistencies PDF featuring all of our current concerns
> *Note*: I realized the "error" in the code came to be during the process of me getting rid of the warnings in the FORTRAN code regarding `DO` loops.

**Comment by EmilSoleymani** [comment]
New typo found in book during explanation of output file formatting. I will add to inconsistencies pdf soon. Page 167:
<img width="471" alt="image" src="https://user-images.githubusercontent.com/97846411/167664242-af625ac3-b98f-4e53-aad0-85824f46aa7f.png">
Highlighted 0 should be 1

**Comment by EmilSoleymani** [comment]
Added in #35

**Comment by smiths** [comment]
Good catch @EmilSoleymani.  You are highlighting the value of active code review for finding errors.  By trying to deduce the purpose of the code you are finding errors.  In our case evidence to support active code review is anecdotal, but controlled experiments have revealed their value, as shown in [KellyAndSheppard2000](https://gitlab.cas.mcmaster.ca/smiths/pub/-/blob/master/KellyAndShepard2000.pdf), where inspectors were asked to create documentation.
